src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -45,15 +45,15 @@ def _get_login_token(login_server, only_refresh_token=True, repository=None):
     headers = {'Content-Type': 'application/x-www-form-urlencoded'}
     if isinstance(refresh, str):
         content = {
+            'grant_type': 'refresh_token',
             'service': params['service'],
-            'credential_type': 'user',
             'tenant': tenant,
             'refresh_token': refresh
         }
     else:
         content = {
+            'grant_type': 'spn',
```

Comment:
This actually makes sense. I was thinking about different terms in AAD and OAuth2.